src/app/common/ResourceStatus.vue

```diff
@@ -0,0 +1,53 @@
+<template>
+  <div class="resource-status">
+    <div class="resource-status-title">
+      <slot name="icon" />
+
+      <slot name="title" />
+    </div>
+
+    <div class="resource-status-container">
+      <span v-if="props.online !== null">{{ props.online }}</span><span
+        v-if="props.online !== null"
+        class="status-separator"
+      >/</span><span class="status-total">{{ props.total }}</span>
+    </div>
+  </div>
+</template>
+
+<script lang="ts" setup>
+const props = defineProps({
+  total: {
+    type: Number,
+    required: true,
+  },
+
+  online: {
+    type: Number,
+    default: null,
+  },
+})
+</script>
+
+<style lang="scss" scoped>
+.resource-status>*+* {
```

Comment:
just FYI: it's all elements that immediately follow another element that are immediately child of `resource-status`